C-Sharp/VectorDistanceCalculation/PE11VectorDistanceCalculation/Menu.cs

```diff
@@ -10,23 +10,31 @@ class Menu
     {
         private static int selection;
         private static int[] validSelections = { 1, 2, 3 };
+        //Have you guys gone over unit tests?
+        //I've seen a few bugs in your code but unit tests will surface those bugs
```

Comment:
We have briefly gone with unit tests in the ASP.NET textbook using Moq, but it's a little too hard to understand...  I am curious about the bugs, though!

Reply:
That's why you write unit tests. So you can find the bugs before you ship your product.

Reply:
Makes sense!  I just need to get used to writing unit tests.